### DISCUSSION TITLE
How to send clickable link in chat?

### DISCUSSION THREAD

**Discussion by AntonIXO** [Minestom/Minestom#2108] [Help] [answered]
I am replying to command execution with "sender.sendMessage("Done! " + url + builAddr);" and wanna make printed url clickable. How to do this?

**Comment by vearendev** [comment, marked as answer]
`Component component = Component.text("click me").clickEvent(ClickEvent.openUrl("https://minestom.net/"));`
`player.sendMessage(component);`

**Reply by zax71** [reply]
This is done using the [Adventure](https://docs.advntr.dev/text.html) library that is built in to Minestom.

**Reply by Tofaa2** [reply]
akternatively you can use MiniMessage by adventure

So
```
MiniMessage minimessage = MiniMessage.miniMessage();

Component c = minimessage.deserialize("<red><click:open_url:`https://minestom.net`>Hello World");
```
